### DISCUSSION TITLE
DIANN picks wrong peaks for quantification in some samples

### DISCUSSION THREAD

**Discussion by tomvaiuw** [vdemichev/DiaNN#869] [Q&A]
Hi Vadim et al,
We have been exploring to use DIANN to quantify large set of samples and we ran into an issue where even though we provide the DIANN with a highly curated spectral library with verified RT in some samples an incorrect small peak is selected even though it is further away from library RT than correct peak. Few examples comparing the DIANN with Skyline (and using Skyline to visualize the data) are attached. What is shown are digestion (DQC) and technical (TQC) replicates so the samples are essentially identical for each of DQC and TQC, respectively. Any advice would be greatly appreciated. We have over 500 samples that we are trying to quantify. 

Thanks a lot,

Tomas Vaisar
UW, Seattle

[S198_Search1_DIAnn_vs_Skyline_DQC_TQC.pptx](https://github.com/vdemichev/DiaNN/files/13395310/S198_Search1_DIAnn_vs_Skyline_DQC_TQC.pptx)

**Comment by vdemichev** [comment]
Hi Tomas,

Both peaks show similar number of fragments detected, at least with fragments visualised. DIA-NN places little to know value in signal intensity, with co-detection of fragments being the main metric - such an apporach makes a lot of sense for DIA in contrast to DDA. Here we probably have two different peptidoforms, with one either more abundant or ionising better, mistakes in distinguishing those are inherent to DIA (note that the output filtered at 1% FDR has much greater peptidoform FDR for any DIA tool). MBR alleviates picking different peptidoforms across different runs, was it enabled in DIA-NN?

Best,
Vadim

**Comment by tomvaiuw** [comment]
Hi Vadim,

Thanks a lot for the information. I understand how the DIA works, yet it is problematic when a wrong peak gets integrated even if a much larger correct peak is closer to expected retention time (from the library). We are finding this is actually relatively frequent (well frequent enough to be frustrating when precise quantification is of interest with large number of samples to make manual validation very time consuming).
We have tried to use MBR, but the problem we ran into is that it alters our spectral library (returning into it ions we eliminated in our master spectral library). The way we do the analysis is – we process GPF runs through Fragpipe/MSFragger to generate the spectral/chromatogram library. We then semi-manually curate the library through Skyline as we found that MSFragger pipeline to generate the spectral library suffers from similar woes – picking peak at wrong RT (low abundance peptidoform) – I discussed this with Alexey and it is a game of numbers where infinitesimally small difference in probability  (e.g. 0.9998 vs 0.9999) results in preference of the smaller peak (they will try to do something about this). We then eliminate b ions, and y1, y2 ions from the library (highly prone to interferences) and correct the RT to make a master corrected spectral library. Then we run DIANN with this library to do the quantification of the actual samples. We are testing with about 60, but eventually we are quantifying nearly 700 samples.
We have tested Skyline with mProphet, Encyclopedia and DIANN and we are finding that DIANN gives us the most consistent results, yet it still produces these spurious results.
We have now tested the most recent DIANN beta version 27, and the v.1.8 with and without MBR.  The beta version without MBR actually did better than with MBR no couple peptides we look at while MBR did better on the third peptide. We also tried the experimental parameter  for cross-run normalization RT and Signal, but did not get much improvement.
While in grand scheme of things this might not be a major issue for proteins with many peptides, we have some 5-10 kDa proteins that provide few proteotypic peptides and there it becomes an issue.
I get your point about DIA preferring agreement between fragments’ chromatographic peaks over intensity however, to me it would make sense to put more weight on it in a second pass to improve the reliability of quantification.

Any suggestion you might have would be greatly welcome.

Best regards,
Tomas

**Comment by vdemichev** [comment]
Hi Tomas,

We a GPF-based lib, indeed MBR will work detrimentally.
One note: pure lib-free analysis of DIA-NN is likely to result in signficantly more proteins identified in the 'real' heterogeneous experiment. It's definitely worth trying it with beta 27, to see how it performs also in terms of peptidoform misidentifications.

Also, with any library, what DIA-NN does is finding the peak best matching to the library entry, so any error in the library will propagate to the DIA-NN search.

Best Regards,
Vadim

**Comment by tomvaiuw** [comment]
Thanks Vadim,
We are pretty certain that our library is correct, that’s why we basically manually validated retention times visually in Skyline. And we are not concerned about more protein identifications and peptidoforms as we’ve run the search multiple ways (different search engines) and came to a consensus library. Our concern is reliable detection of the peptidoforms in the library in all samples. Unfortunately, all programs (Encyclopedia, Skyline, DIANN) we have tried “misfire” on some samples, but DIANN seemed to perform better than the other so we’ve been trying to figure out whether there are parameters that would make it even more reliable.

Best,
Tomas

**Comment by tomvaiuw** [comment]
Vadim,
Thanks a lot once more for the advice. We have tried to run pure lib-free analysis of DIA-NN and it did not perform better than any of our previous tests. In fact, in case of one of the peptides we monitored for the “small” peak peptidoform, it invariably chose the wrong (small) peak. In the other cases it alternatively chose correct peak and the “small” peak across the set of about 70 samples.

Best regards,
Tomas

**Reply by JunYao-mynamesofewletters** [reply]
Hi, Tomas! Have you checked the LC performance ? Maybe Frodo and Gandolf were both genuine peptide signal ?

**Comment by tomvaiuw** [comment]
I believe that the two peaks are legit peptide isomers and that the LC separation works fine. However, they should either both get quantified, or one of them should be quantified consistently. Currently it is random – in some sample it is Frodo and another Gandalf….

Tomas
